server/src/lib.rs

```diff
@@ -269,11 +269,15 @@ pub fn get_ip(conn_info: &ConnectionInfo) -> String {
     .to_string()
 }
 
+pub fn is_valid_register_username(name: &str) -> bool {
```

Comment:
Sounds wrong grammatically, maybe `is_valid_username()` (but also this doesnt really matter).

Reply:
K cool, I changed this now.